roles/nv_gpu_install_from_commit/files/operator_image_builder_pod.yml

```diff
@@ -25,7 +25,7 @@ spec:
       - name: IMAGESTREAM_TAG
         value: "{{ gpu_operator_image_tag }}" # set by `build.yaml` ansible script
       - name: BUILDER_FROM_IMAGE
-        value: "quay.io/openshift-psap/golang:1.13" # avoid using docker.io and its quotas...
+        value: "quay.io/openshift-psap/golang:1.15" # avoid using docker.io and its quotas...
```

Comment:
4.8 is 1.16, maybe we want to consider it

Reply:
hum, I don't really know, I just used whatever they use upstream,
what do you thing?

Reply:
1.15 is ok for now, 4.8 is 2 months in the horizon. we can bump to 1.16 later on